Lib/distutils/sysconfig.py

```diff
@@ -13,543 +13,61 @@
-            cpp = cpp + ' ' + os.environ['CPPFLAGS']
-            cflags = cflags + ' ' + os.environ['CPPFLAGS']
-            ldshared = ldshared + ' ' + os.environ['CPPFLAGS']
-        if 'AR' in os.environ:
-            ar = os.environ['AR']
-        if 'ARFLAGS' in os.environ:
-            archiver = ar + ' ' + os.environ['ARFLAGS']
-        else:
-            archiver = ar + ' ' + ar_flags
-
-        cc_cmd = cc + ' ' + cflags
-        compiler.set_executables(
-            preprocessor=cpp,
-            compiler=cc_cmd,
-            compiler_so=cc_cmd + ' ' + ccshared,
-            compiler_cxx=cxx,
-            linker_so=ldshared,
-            linker_exe=cc,
-            archiver=archiver)
-
-        compiler.shared_lib_extension = shlib_suffix
-
-
-def get_config_h_filename():
-    """Return full pathname of installed pyconfig.h file."""
-    if python_build:
-        if os.name == "nt":
-            inc_dir = os.path.join(_sys_home or project_base, "PC")
-        else:
-            inc_dir = _sys_home or project_base
-    else:
-        inc_dir = get_python_inc(plat_specific=1)
-
-    return os.path.join(inc_dir, 'pyconfig.h')
-
-
-def get_makefile_filename():
-    """Return full pathname of installed Makefile from the Python build."""
-    if python_build:
-        return os.path.join(_sys_home or project_base, "Makefile")
-    lib_dir = get_python_lib(plat_specific=0, standard_lib=1)
-    config_file = 'config-{}{}'.format(get_python_version(), build_flags)
-    if hasattr(sys.implementation, '_multiarch'):
-        config_file += '-%s' % sys.implementation._multiarch
-    return os.path.join(lib_dir, config_file, 'Makefile')
+from functools import partial
 
+from .errors import DistutilsPlatformError
 
+from sysconfig import _PREFIX as PREFIX
+from sysconfig import _BASE_PREFIX as BASE_PREFIX
+from sysconfig import _EXEC_PREFIX as EXEC_PREFIX
+from sysconfig import _BASE_EXEC_PREFIX as BASE_EXEC_PREFIX
+from sysconfig import _PROJECT_BASE as project_base
+from sysconfig import _PYTHON_BUILD as python_build
+from sysconfig import _CONFIG_VARS as _config_vars
+from sysconfig import _init_posix as sysconfig_init_posix
+from sysconfig import parse_config_h as sysconfig_parse_config_h
+from sysconfig import _parse_makefile as sysconfig_parse_makefile
+
+from sysconfig import _init_non_posix
+from sysconfig import _is_python_source_dir
+from sysconfig import _sys_home
+
+from sysconfig import _variable_rx
+from sysconfig import _findvar1_rx
+from sysconfig import _findvar2_rx
+
+from sysconfig import build_flags
+from sysconfig import customize_compiler
+from sysconfig import expand_makefile_vars
+from sysconfig import is_python_build
+from sysconfig import get_config_h_filename
+from sysconfig import get_config_var
+from sysconfig import get_config_vars
+from sysconfig import get_makefile_filename
+from sysconfig import get_python_inc
+from sysconfig import get_python_version
+from sysconfig import get_python_lib
```

Comment:
I'm not personally opposed to putting them on separate lines, but you may get feedback that it should all be in a single `from...import` statement.

Reply:
It made it easier for me when I was working on it part by part. I'll merge them into one.